i18n/fr/docusaurus-plugin-content-docs/current/full-node/Full-Node-More.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position : 40
+titre : Plus d'information et dépannage
```

Comment:
There should be no spaces before colons in this section. Please apply this in other pages as well

```suggestion
sidebar_position: 40
titre: Plus d'information et dépannage
```